common/src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/projectimportexport/ProjectImporter.kt

```diff
@@ -136,17 +143,8 @@ class ProjectImporter @Inject constructor(
         fileReader: IFileReader,
         metadata: ResourceMetadata,
         manifestProject: Project,
-        manifestSources: Set<Source>
+        sourceCollection: Collection
     ) {
-        val existingSource = fetchExistingSource(manifestProject, manifestSources)
-        // if any relevant source exists then use it
-        val sourceCollection = if (existingSource == null) {
-            importSources(fileReader)
-            findSourceCollection(manifestSources, manifestProject)
-        } else {
-            existingSource
-        }
-
```

Comment:
this block was moved to the method above ^ so that the project metadata can be constructed with appropriate version, before passing into this method